easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = "PythonPackage"
+
+name = 'HTSeq'
+version = '0.9.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www-huber.embl.de/users/anders/HTSeq/'
+description = """A framework to process and analyze data from high-throughput sequencing (HTS) assays"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['af5bba775e3fb45ed4cde64c691ebef36b0bf7a86efd35c884ad0734c27ad485']
+
+dependencies = [
+    ('numpy', '1.10.1', versionsuffix),
+    ('Pysam', '0.9.1.4', versionsuffix),
+    ('Python', '2.7.12'),
+    ('matplotlib', '1.5.3', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': ["bin/htseq-count", "bin/htseq-qa"],
+    'dirs': ["lib/python%(pyshortver)s/site-packages/%(name)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg"],
+}
+
+sanity_check_commands = [('htseq-count', '--help')]
```

Comment:
@boegel I changed it as proposed but the sanity check still completes succesfully and when the command is executed after the module is loaded it still fails?